prow/jobs/jobs.yaml

```diff
@@ -1,6 +1,41 @@
 # Autogenerated. Do NOT update Manually.
-# Last generated on 2022-11-10 17:43:10.641074.
+# Last generated on 2022-11-22 15:28:14.463413.
 periodics:
+- name: label-sync
+  labels:
+    app: label-sync
+  decorate: true
+  annotations:
+    description: Runs label_sync to synchronize GitHub repo labels with the label config defined in label_sync/labels.yaml.
+  agent: kubernetes
+  spec:
+    containers:
+    - name: label-sync
+      image: gcr.io/k8s-prow/label_sync:v20220330-40eb179576
+      command:
+      - label_sync
+      args:
+      - --config=/etc/config/labels.yaml
+      - --confirm=true
+      - --orgs=aws-controllers-k8s
+      - --token=/etc/github/oauth
+      - --github-endpoint=http://ghproxy
+      - --github-endpoint=https://api.github.com
+      - --debug
```

Comment:
Do you want to keep this `--debug` here?

Reply:
Left on purpose to see how to first job execution will go. Planning on removing this flag sometime in Q1.